definitions/npm/winston_v3.x.x/flow_v0.34.x-/winston_v3.x.x.js

```diff
@@ -111,7 +111,8 @@ declare module "winston" {
   declare export type DefaultLogger = $winstonDefaultLogger;
   declare export type Container<T: Levels> = $winstonContainer<T>;
 
-  declare module.exports: $winstonDefaultLogger & {
+  declare module.exports: {
+    ...$Exact<$winstonDefaultLogger>,
```

Comment:
Does it still work with `commonjs` though?

Reply:
Hm, I don't see why it shouldn't.  Added an example test case:

https://github.com/flow-typed/flow-typed/pull/3447/commits/e1bae314782b7c675f98e565d8022d631db80d71

Reply:
Thanks!